ide/src/main/java/com/devonfw/tools/ide/log/IdeLogger.java

```diff
@@ -238,13 +238,13 @@ default void error(Throwable error, String message) {
       }
       return;
     }
-    boolean traceEnabled = trace().isEnabled();
+    boolean traceEnabled = true; // trace().isEnabled();
     boolean debugEnabled = debug().isEnabled();
-    if (hasMessage && !debugEnabled) {
-      // assuming that if debug is disabled then also trace is disabled so we ignore error
-      error(message);
-      return;
-    }
+    // if (hasMessage && !debugEnabled) {
+    // // assuming that if debug is disabled then also trace is disabled so we ignore error
+    // error(message);
+    // return;
+    // }
```

Comment:
Can be removed?

Reply:
Thanks. Done: https://github.com/devonfw/IDEasy/commit/e425f8f6ecdfc93fef74f3a89479a8ee14579d73